lib/cog/chat/slack/connector.ex

```diff
@@ -323,6 +332,44 @@ defmodule Cog.Chat.Slack.Connector do
     end
   end
 
+  defp normalize_channel_id(:by_id, id = "C" <> _, slack) do
```

Comment:
Not if it isn't an id. The `"C"` verifies that what was passed was an id. If it isn't an id, then we assume it's the name of a channel and do a lookup.